src/field_effect.c

```diff
@@ -3292,7 +3303,7 @@ static void FlyOutFieldEffect_FlyOffWithBird(struct Task *task)
         struct ObjectEvent *objectEvent = &gObjectEvents[gPlayerAvatar.objectEventId];
         ObjectEventClearHeldMovementIfActive(objectEvent);
         objectEvent->inanimate = FALSE;
-        objectEvent->hasShadow = FALSE;
+        objectEvent->noShadow = TRUE; // TODO: Make shadow smaller instead of disappearing completely ?
```

Comment:
I don't think we should preserve TODOs like this but rather open an issue so we don't forget (or just leave it to merrp)

Reply:
Removing the comment locally